lib/find-readme-file.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
```

Comment:
this logic follows GitHub's precedence rules for readmes. It should probably be broken out into its own npm package eventually.